src/index.html

```diff
@@ -1,14 +1,70 @@
+
 <!doctype html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap">
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+          class="logo__image"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
```

Comment:
```suggestion
          <li class="nav__item">
            <a href="#" class="nav__link is-active">
              Apple
            </a>
          </li>
```
Such formatting will help to increase code readability 
![image](https://github.com/mate-academy/layout_moyo-header/assets/96409563/fa512fc0-48d1-4f55-9e05-44fe906bfd3f)